edi-tools/modules/codegen/rest_connector_gen.bal

```diff
@@ -31,7 +36,11 @@ service /${re.replaceAll(libName,"_")}Parser on new http:Listener(restConnectorP
         anydata target = check fromEdiString(ediData, ediTypeName);   
         return target.toJson();
     }
-
+    # Resource to convert Ballerina record to EDI string.
+    # 
+    # + ediType - EDI type name.
+    # + jsonData - Ballerina record as JSON.
+    # + return - EDI string.
```

Comment:
Remove full stop at the end.

Reply:
Changed with 1330f90e144a8514f5a75f7fd1d041a44035e7ba.